app/views/tags/edit.html.erb

```diff
@@ -16,16 +16,6 @@
         tags: @revision.tags,
         contextual_guidance_id: "#{tag_field.id}-guidance" %>
     </div>
-    <% if @document.document_type.guidance_for(tag_field.id) %>
-    <div class="govuk-grid-column-one-third">
-      <%= render "components/contextual_guidance", {
-        id: "#{tag_field.id}-guidance",
-        title: @document.document_type.guidance_for(tag_field.id).title
-      } do %>
-        <p class="govuk-body"><%= @document.document_type.guidance_for(tag_field.id).body %></p>
```

Comment:
How sure are we that we're not needing the guidance for this (maybe we've lost it along the way)?
I remember I did some work on the guidance component a while back just to make it work with autocomplete.

Reply:
So I can see it's removed for tags, to be replaced with hint text, while for something like contacts it's just not applicable. It looks like it would be pretty simple to add it in again, if we did need to support it, but I thought this was a good opportunity to clean up. Do you think we should keep it?

Reply:
Sure, if we're providing the hint text I'm happy to drop it. The less we're using the contextual guidance aside from the edit page the better.

Reply:
👍